modal/eggs_modal.php

```diff
@@ -64,7 +64,25 @@ class="form-control text-center">
             <input type="radio" name="clean" id="clean_1" value="clean_1" <?php echo $checked1; ?>> Yes
         </label>
     </div>
-
+    <hr>
+    <?php if (isset($allowed_templates)) {
+        echo '<div class="form-row align-items-center">
+            <div class="col-sm-12 my-1">
+                <div class="btn-group btn-group-toggle" data-toggle="buttons">
+                    <div class="input-group">
+                        <div class="input-group-prepend">
+                            <div class="input-group-text">Template</div>
+                        </div>
+                    </div>';
+                    foreach ( $allowed_templates as $key => $name ) {
+                        echo '<label class="btn btn-secondary">';
+                        echo '<input type="radio" name="template" id="' . $key . '" value="' . $key . '" ' . (($key == $row['template']) ? 'checked' : '') . '>';
```

Comment:
@bbdoc  hope you don't mind that i used a slightly different approach on the checked tag on a button. using ternary line allowed me to use less code.